session.go

```diff
@@ -625,12 +624,15 @@ func (s *session) handleCloseError(closeErr closeError) error {
 }
 
 func (s *session) processTransportParameters(params *handshake.TransportParameters) {
-	s.remoteIdleTimeout = params.IdleTimeout
-	s.flowControlManager.UpdateTransportParameters(params)
+	s.peerParams = params
 	s.streamsMap.UpdateTransportParameters(params)
```

Comment:
I must have missed this earlier, but I would suggest not passing the params struct here too.

Reply:
Fixed in #879.